### ISSUE TITLE
Vendor our dependencies

### ISSUE THREAD

**Issue by karlek** [mewkiz/flac#24]
Since we depend on a few libraries, and a few projects depends on us, it would be high time to start vendoring our dependencies.

The only problem is in deciding which dependancy manager to use. I argue that we first use `dep`, until `vgo` leaves the prototype stage. However, bleeding edge is also fun in itself.

What's your take on this @mewmew?

**Comment by mewmew** [comment]
Since we haven't been using any dependency management up until now, we don't have any users depending on us *having figured it out* yet.

Thus, I'd say, lets bleed a bit!

**Comment by karlek** [comment]
Well, it's bleeding a bit too much for me haha
```fish
$ vgo; cat /usr/lib/go/VERSION
vgo requires Go 1.10 but VGOROOT=/usr/lib/go is not a Go 1.10 source tree
go1.10.2⏎
```

**Comment by mewmew** [comment]
Haha :) Using vgo on Arch is a blood bath.

Upstream issue golang/go#24969.

Try
```bash
sudo mkdir /usr/lib/go/api
sudo touch /usr/lib/go/api/go1.10.txt
```
and see if that works.

If so, the AUR package of Arch has to be updated to include that file.

**Comment by karlek** [comment]
Yeah, that worked! I actually use the `community/go` package :cake:

**Comment by mewmew** [comment]
What version of vgo where you using, i.e. what rev (git revision of golang.org/x/tools)?

**Comment by karlek** [comment]
```
$ vgo version
go version go1.10.2 linux/amd64 vgo:2018-02-20.1
```

**Comment by mewmew** [comment]
Great. Reported upstream to Arch: https://bugs.archlinux.org/task/58752

**Comment by karlek** [comment]
Erh, I'm sorry brother. I use `extra/go` :cry:

**Comment by mewmew** [comment]
No worries, added a comment about extra/go: https://bugs.archlinux.org/task/58752#comment169779

**Comment by mewmew** [comment]
Fixed with #27.
